### PR TITLE
Github module: use githubProfile from LDAP if enabled by parameter

### PR DESCRIPTION
**Pull request by milescrabill**
updated mozldap, ready to merge.

### CODE REVIEW

File: modules/github/README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Config explanations:
       - credentials:
             - oauthtoken: a Github OAuth2 or Personal Access Token
       - parameters:
+            - useldapgithubaccountmapping: indicates whether to use the Github field in LDAP as opposed to the uidmap in config to map LDAP usernames to Github usernames
```
Comment (jvehent) [inline modules/github/README.md]: I'm not sure we should do an either/or behavior here. Userplex is an LDAP-to-many multiplexer, and I think we should require the data to be in LDAP. If a users don't have their github usernames in LDAP, we should just remove them from the org.

Given this, I'm not sure we need to use the uidmap at all in this module.

**Review by jvehent — COMMENTED** [review]
The code is fine, but I'm not sure the feature is something we want.

File: modules/github/README.md (hunk 1 above)
Comment (milescrabill) [reply modules/github/README.md]: Yeah, I erred on the side of caution - didn't introduce divergent functionality right off the bat. I agree. Fixed.

**Review by jvehent — APPROVED** [review]
r+

**Pull request merged**